service/chat.py

```
@@ -37,6 +43,53 @@ async def completions(conversation_body:str, model_dict, api_version:str = None)
+    # Craft a special instruction if tools are being used
+    if has_tools and use_tools:
+        # Create a system instruction that explains the tools
+        tool_instructions = "You have access to the following tools:\n"
+        
+        for tool in conversation.tools:
+            if hasattr(tool, 'type') and tool.type == "function":
+                if hasattr(tool, 'function'):
+                    func = tool.function
+                    tool_instructions += f"- {func.name}: {func.description}\n"
+            elif isinstance(tool, dict) and tool.get('type') == "function":
+                if 'function' in tool:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)